<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWindow</class>
 <widget class="QWidget" name="RegisterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RegisterWindow</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">#widget{
border-image: url(:/my_image/timgu.jpg);
}</string>
     </property>
     <widget class="QLineEdit" name="usernamelineEdit">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>90</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/image/login/loging_button_bj.png);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="registerushButton">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>310</y>
        <width>89</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>注册</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="telephonelineEdit">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>240</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/image/login/loging_button_bj.png);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="passwordlineEdit">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>170</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/image/login/loging_button_bj.png);</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>90</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
border-image: url(:/image/login/loging_button_bj.png);
font: 10pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>用户名</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>170</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/image/login/loging_button_bj.png);
color: rgb(0, 0, 0);
font: 10pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>密码</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>240</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 10pt &quot;Arial&quot;;
border-image: url(:/image/login/loging_button_bj.png);</string>
      </property>
      <property name="text">
       <string>电话</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>71</width>
        <height>281</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);
font: 28pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>欢
迎
使
用
！</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
